blogs/2022/05/18/dev-container-cli.md

```diff
@@ -0,0 +1,131 @@
+---
+Order: 73
+TOCTitle: Dev container CLI
+PageTitle: Development Container CLI
+MetaDescription: Provide a consistent development environment anywhere using the development container CLI.
+Date: 2022-05-18
+Author: Brigit Murtaugh
+---
+
+# The dev container CLI
+**A consistent environment anywhere with the dev container CLI**
+
+May 18, 2022 by Brigit Murtaugh, [@BrigitMurtaugh](https://twitter.com/BrigitMurtaugh)
+
+## TLDR;
+
+The dev container teams at Microsoft and GitHub are excited to release a CLI as part of the new [Development Containers Specification](https://github.com/devcontainers/spec). We’d love your feedback in [this issue](https://github.com/devcontainers/cli/issues/7), or new issues and PRs in the repo.
```

Comment:
Maybe mention it's an open source CLI?

Reply:
Great point, added "release an **open source** CLI..."